versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5804 @@
+```
+
+##### Equality and Inequality Comparison of Optional Types
+
+The equality and inequality operators are exceptions to the general rules on [coercion of optional types](#coercion-of-optional-types). Either or both operands of an equality or inequality comparison can be optional, considering that `None` is equal to itself but no other value.
+
+```wdl
+Int i = 1
+Int? j = 1
+Int? k = None
+
+i == j  # true
+i == k  # false
+j != k  # true
+k == None  # true
+```
+
+#### Operator Precedence Table
+
+| Precedence | Operator type         | Associativity | Example              |
+|------------|-----------------------|---------------|----------------------|
+| 11         | Grouping              | n/a           | (x)                  |
+| 10         | Member Access         | left-to-right | x.y                  |
+| 9          | Index                 | left-to-right | x[y]                 |
+| 8          | Function Call         | left-to-right | x(y,z,...)           |
+| 7          | Logical NOT           | right-to-left | !x                   |
+|            | Unary Negation        | right-to-left | -x                   |
+| 6          | Multiplication        | left-to-right | x*y                  |
+|            | Division              | left-to-right | x/y                  |
+|            | Remainder             | left-to-right | x%y                  |
+| 5          | Addition              | left-to-right | x+y                  |
+|            | Subtraction           | left-to-right | x-y                  |
+| 4          | Less Than             | left-to-right | x<y                  |
+|            | Less Than Or Equal    | left-to-right | x<=y                 |
+|            | Greater Than          | left-to-right | x>y                  |
+|            | Greater Than Or Equal | left-to-right | x>=y                 |
+| 3          | Equality              | left-to-right | x==y                 |
+|            | Inequality            | left-to-right | x!=y                 |
+| 2          | Logical AND           | left-to-right | x&&y                 |
+| 1          | Logical OR            | left-to-right | x\|\|y               |
+
+#### Member Access
+
+The syntax `x.y` refers to member access. `x` must be a call in a workflow, or a struct or object value. A call can be thought of as a struct where the members are the outputs of the called task.
+
+```wdl
+call foo
+String x = foo.y
+
+Struct z
+String a = z.b
+```
+
+#### Ternary operator (if-then-else)
+
+This is an operator that takes three arguments: a condition expression, an if-true expression, and an if-false expression. The condition is always evaluated. If the condition is true then the if-true value is evaluated and returned. If the condition is false, the if-false expression is evaluated and returned. The if-true and if-false expressions must return values of the same type, such that the value of the if-then-else is the same regardless of which side is evaluated.
+
+Examples:
```

Comment:
These examples are not feeling useful to me.  They need more context to really understand what they are trying to tell the reader.  I don't think you've introduced the runtime section yet.  Also, making your examples closer to valid WDL would be good.  The `Boolean morning = ...` especially - what are you intending here?  What actual example could make this more clear?  Maybe put 
```
# in input to workflow, the value of morning is set to either true or false
Boolean morning = true # or false
String greeting = "good " + if morning then "morning" else "afternoon"
# Runtime evaluation if morning is true
greeting = "good morning"
# Runtime evaluation if morning is false
greeting = "good afternoon"
```

OR perhaps something else.

Reply:
I turned them into task examples